db.go

```diff
@@ -75,12 +75,26 @@ func (self *DB) Query(sql string, vars map[string]any) (any, error) {
 	return self.send("query", sql, vars)
 }
 
+func (self *DB) SchemalessSelect(what string) ([]map[string]interface{}, error) {
+	output, err := self.Select(what)
+	if err != nil {
+		return make([]map[string]interface{}, 0), err
```

Comment:
you can simply `return nil, err` to avoid the allocation

Reply:
Updated